Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -203,7 +203,9 @@ public async void StartRound(bool force = false)
                 RoundLengthMetric.Set(0);
 
                 var playerIds = _playersInLobby.Keys.Select(player => player.UserId.UserId).ToArray();
-                RoundId = await _db.AddNewRound(playerIds);
+                var serverName = _configurationManager.GetCVar(CCVars.ServerId);
```

Comment:
There are multiple problems with using this value:
1. This value is intended for stuff like client-side storage of rules and changelog info. It is NOT unique between our servers.
2. The default value for this (because yes, people WILL forget to set this) is different from the default fallback "unknown" value that you put in the `server` table.

Make a new one instead that we can keep internally in our infrastructure instead of being public facing like this.

Reply:
I made an new one and named it adminlogs.server_name. The default value is "unknown".